### DISCUSSION TITLE
Cannot publish subpage

### DISCUSSION THREAD

**Discussion by ptanasri** [oleeskild/obsidian-digital-garden#212] [Q&A]
Hi, I have some issues with the subpages. They show up on the sidebar and can be accessed alright i.e. the url is clickable and points correctly to the subpages.

However, when I put those subpages ( as LINKS using [[ ) on the homepage, they are not accessible.

All the urls (containing these "published" subpages) will open up to to a new webpage(s) but with errors saying "this note is probably not made public". 

Can anyone help please. Thank you.

**Comment by shinanqu** [comment]
Can you post the link?
The issue is that the links are wrong, do I understand that correctly? In the sidebar, the links are correct, and the subpages are accessible normally. But when you write normal text and implement links with [[ then the links are wrong?
Or the final pages are inaccessible at all on the homepage?
I had some issues with pages that were inaccessible when the front matter had some issues. In these cases, the pages were not published at all. Make sure that the pages are listed as Published in the Publication Center.

**Reply by ptanasri** [reply]
thank you for your reply. let me give the url so you can see it more clearly

the link (in the sidebar) is working ( https://xyz.netlify.app/veggy-midstream/erp%20is%20huge./)
but the link in the subpage that I put in ob under the command [[  is not working  i.e. the url shows up as " https://xyz.netlify.app/veggy-midstream/indexpage/ERP%20is%20huge..md " eventhough the content is working alright inside the obsidian application.

this is another example.
https://xyz.netlify.app/veggy-midstream/key%20action%20plans/ (link working)
https://xyz.netlify.app/veggy-midstream/indexpage/Key%20Action%20Plans.md (link not working)

( i think there is something wrong with the translation into the url )

**Reply by ptanasri** [reply]
sorry I replace the text with xyz (as some info in the link has some personal data).

**Reply by shinanqu** [reply]
For me, no provided link was working. Neither
https://xyz.netlify.app/veggy-midstream/erp%20is%20huge./
https://xyz.netlify.app/veggy-midstream/key%20action%20plans/
nor the main page
https://xyz.netlify.app/veggy-midstream/

It's interesting that the working links have the .md ending. My homepage has no .md endings at all. And your first link ends with a dot: %20huge.

Sadly I can not say what the issue is. My links are working fine, even when I use spaces in the title.

What you can try is the dg-permalink
dg-permalink: "mynote"
https://dg-docs.ole.dev/advanced/note-specific-settings/

And you are using Netlify. I have the impression that Digital Garden has moved to Vercel. I never used Netlify, so I don't know if there were issues with Netlify. I checked an example Netlify Digital Garden, and he doesn't use any spaces in his URLs, only hyphens.
https://edav-garden.netlify.app

**Reply by ptanasri** [reply]
i have tried to the permalink trick. again, the link is working with the permalink LOCKED as url as well. but sadly, the link (in the subpage) is still not working i.e. its url stubbornly refused to change to the permalink. 

anyway, thanks for your inputs. I will keep tyring.

**Reply by shinanqu** [reply]
You have a permalink but the URL is not changing?
Well, I had an issue with a broken front matter. Some other parts of the front matter caused issues and so Digital Garden ignored my permalink. If the permalink is ignored, can you try to clean up the front matter. Just keep the two lines in it as a first step.

dg-publish: 
dg-permalink: 

If it's still not working, then I can not help, other than try to reinstall everything again and perhaps use Vercel (though Netlify should usually work fine, I guess).
I wish you good luck!
